internal/tiltfile/encoding/yaml.go

```diff
@@ -90,6 +134,28 @@ func encodeYAML(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tup
 	return tiltfile_io.NewBlob(ret, "encode_yaml"), nil
 }
 
+func encodeYAMLStream(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
```

Comment:
conversely, should it be an error if you try to call encodeYAML on a list?

Reply:
nope - a list is a valid yaml document, e.g. `encode_yaml(["foo", "bar"]) == "- foo\n- bar\n"`